packages/cardano-services/src/Metadata/types.ts

```diff
@@ -1,11 +1,24 @@
 import { Cardano } from '@cardano-sdk/core';
 
-export interface MetadataService {
+export interface DbSyncMetadataService {
```

Comment:
This change is a (breaking) `refactor`, best committed separately.

Reply:
Split into a separate breaking change commit: 28ac6c3ccea07afdbdac16f2cf0bfb5b86277064